packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -492,20 +491,34 @@ class _CupertinoAlertDialogState extends State<CupertinoAlertDialog> {
   }
 }
 
-/// Rounded rectangle surface that looks like an iOS popup surface, e.g., alert dialog
-/// and action sheet.
+/// An iOS-style component for creating modal overlays like dialogs and action
+/// sheets.
 ///
-/// A [CupertinoPopupSurface] can be configured to paint or not paint a white
-/// color on top of its blurred area. Typical usage should paint white on top
-/// of the blur. However, the white paint can be disabled for the purpose of
-/// rendering divider gaps for a more complicated layout, e.g., [CupertinoAlertDialog].
-/// Additionally, the white paint can be disabled to render a blurred rounded
-/// rectangle without any color (similar to iOS's volume control popup).
+/// By default, [CupertinoPopupSurface] generates a rounded rectangle surface
+/// that applies two effects to the background content:
+///
+///   1. Background filter: Saturates and then blurs content behind the surface.
+///   2. Overlay color: Covers the filtered background with a transparent
+///      surface color. The color adapts to the CupertinoTheme's brightness:
+///      light gray when the ambient [CupertinoTheme] brightness is
+///      [Brightness.light], and dark gray when [Brightness.dark].
+///
+/// The blur strength can be changed by setting [blurSigma] to a positive value,
+/// or removed by setting the [blurSigma] to 0.
+///
+/// The saturation effect can be removed for debugging by setting
+/// [debugIsVibrancePainted] to false. The saturation effect is not supported on
+/// the html web renderer and will not be applied regardless of the value of
```

Comment:
```suggestion
/// web with the skwasm renderer and will not be applied regardless of the value of
```

The HTML renderer has been deprecated and removed https://docs.flutter.dev/platform-integration/web/renderers so we only mention skwasm here.